packages/sdk/src/Incentives/ERC20PeggedIncentive.ts

```diff
@@ -503,7 +503,7 @@ export class ERC20PeggedIncentive extends DeployableTarget<
    *
    * @public
    * @param {Hex} claimData
-   * @returns {BigInt} Returns the reward amount from a claim data payload
+   * @returns {Promise<bigint>} Returns the reward amount from a claim data payload
```

Comment:
i don't really support changing this interface for all incentives to support one unique incentive